pkg/scheduler/apis/config/validation/validation_test.go

```diff
@@ -225,102 +226,202 @@ func TestValidateKubeSchedulerConfigurationV1beta2(t *testing.T) {
 		"bad-parallelism-invalid-value": {
 			expectedToFail: true,
 			config:         invalidParallelismValue,
-			errorString:    "should be an integer value greater than zero",
+			wantErrs: field.ErrorList{
+				&field.Error{
+					Type:  field.ErrorTypeInvalid,
+					Field: "parallelism",
+				},
+			},
 		},
 		"bad-resource-name-not-set": {
 			expectedToFail: true,
 			config:         resourceNameNotSet,
-			errorString:    "resourceName is required",
+			wantErrs: field.ErrorList{
+				&field.Error{
+					Type:  field.ErrorTypeInvalid,
```

Comment:
uhm... we should change this to ErrorTypeRequired, but probably a separate PR, as it's not scheduler code